scripts/Tools/testreporter.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+"""
+
+Simple script to populate CESM test database with test results.  This can be 
+run with out any knowledge of CIME.
+
+"""
+
+import os
```

Comment:
Now using Tools.standard_script_setup import *
glob isn't a part of the standard script setup.